models/stdlib.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "LIonCore_M3_Metamodel",
+    "id": "LIonCore_stdlib",
+    "properties": {
+      "LIonCore_M3_Metamodel_qualifiedName": "LIonCore.stdlib"
+    },
+    "children": {
+      "LIonCore_M3_Metamodel_elements": [
+        "LIonCore_stdlib_String",
+        "LIonCore_stdlib_boolean",
+        "LIonCore_stdlib_int",
+        "LIonCore_stdlib_JSON"
+      ]
+    },
+    "references": {
+      "LIonCore_M3_Metamodel_dependsOn": []
+    }
+  },
+  {
+    "type": "LIonCore_M3_PrimitiveType",
+    "id": "LIonCore_stdlib_String",
```

Comment:
Why are ids of `String` and `JSON` uppercase vs `boolean` and `int` lowercase?

Reply:
According to https://github.com/LIonWeb-org/organization/blob/main/lioncore/metametamodel.adoc#pre-defined-primitive-types it should be `String`, `Boolean`, and `Integer`, indeed. That URL doesn't mention JSON, though: I'm OK with deleting it here, but e.g. the Java impl. does have it.

Reply:
Oh, in the interest of meta-circularity: the M3 diagrams should then also use type `Boolean` (for properties), rather than `boolean`, I think.